src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java

```diff
@@ -1,9 +1,23 @@
 package mate.academy.hibernate.relations.util;
 
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
+    private static SessionFactory sessionFactory;
+
+    public HibernateUtil() {
+    }
+
     public static SessionFactory getSessionFactory() {
-        return null;
+        if (sessionFactory == null) {
+            try {
+                sessionFactory = new Configuration().configure().buildSessionFactory();
+            } catch (Exception e) {
+                throw new ExceptionInInitializerError("Initial SessionFactory creation failed: "
+                        + e);
+            }
+        }
+        return sessionFactory;
     }
```

Comment:
You should add a private default constructor to the `HibernateUtil` class in order to prevent creating `HibernateUtil` objects. It's a utility class, and we don't want to create instances of it.